lomond/parser.py

```diff
@@ -136,6 +135,7 @@ def _check_length(pos):
                 ParseError('unexpected eof of file')
             )
 
+        _buffer = self._buffer
```

Comment:
this is to avoid typing self. or does this does something under the hood?

Reply:
It's a micro-optimization. `self._buffer` requires a dict lookup, but a local is faster.